api/db-cdr/prep-tables/make-prep-survey.py

```diff
@@ -0,0 +1,435 @@
+import argparse
+import csv
+import os
+import shutil
+from google.cloud import bigquery
+from google.cloud import storage
+from io import StringIO
+
+# we will increment this as the row number as we write each row in each output file
+id_controlled = 1
+id_registered = 1
+id_all = 1
+
+home_dir = "../csv"
+
+# key = csv prefix, value = csv output file name
+surveys = {"ENGLISHBasics_DataDictionary": "Basics",
+           "ENGLISHHealthCareAccessUtiliza_DataDictionary": "HealthCareAccessUtiliza",
+           "ENGLISHLifestyle_DataDictionary": "Lifestyle",
+           "ENGLISHOverallHealth_DataDictionary": "OverallHealth",
+           "ENGLISHPersonalMedicalHistory_DataDictionary": "PersonalMedicalHistory"}
+# field annotation flags
+annotation_flags = {"REGISTERED_TOPIC_SUPPRESSED": "0",
+                    "REGISTERED_QUESTION_SUPPRESSED": "0",
+                    "REGISTERED_ANSWERS_BUCKETED": "0",
+                    "CONTROLLED_TOPIC_SUPPRESSED": "0",
+                    "CONTROLLED_QUESTION_SUPPRESSED": "0",
+                    "CONTROLLED_ANSWERS_BUCKETED": "0"}
+# these are the column names for the controlled and registered output files
+headers = ['id', 'parent_id', 'code', 'name', 'type', 'min', 'max',
+           'answers_bucketed']
+# these are the headers for outputAll file
+headersAll = ['id', 'parent_id', 'code', 'name', 'type', 'min', 'max',
+              'registered_topic_suppressed', 'registered_question_suppressed',
+              'registered_answer_bucketed', 'controlled_topic_suppressed',
+              'controlled_question_suppressed', 'controlled_answer_bucketed']
+
+# suppression of certain answers from output file
+answerSuppressed = 'CONTROLLED_ANSWER_SUPPRESSED'
```

Comment:
This probably isn't necessary, since it's only referenced once in your file. You could just use 'CONTROLLED_ANSWER_SUPPRESSED' in the code below that references this.